code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -367,32 +367,7 @@
 				if (miming)
 					message = "<B>[src]</B> acts out a scream!"
 				else
-					var/DNA = dna.species.id
-					var/sound = pick('sound/misc/scream_m1.ogg', 'sound/misc/scream_m2.ogg')
-					switch(DNA)
-						if("IPC")
-							sound = "sound/voice/screamsilicon.ogg"
-						if("tarajan")
-							sound = "sound/misc/cat.ogg"
-						if("lizard")
-							sound = "sound/misc/lizard.ogg"
-						if("avian")
-							sound = "sound/misc/caw.ogg"
-						if("skeleton")
-							sound = "sound/misc/skeleton.ogg"
-						if ("moth")
-							sound = "sound/misc/moth.ogg"
-						else
-							if(gender == FEMALE)
-								sound = pick('sound/misc/scream_f1.ogg', 'sound/misc/scream_f2.ogg')
-							if(isalien(src))
```

Comment:
why the fuck was this here if aliens aren't classified as humans,the hell. Just curious